lib/dev3/web/plugs/rate_limit.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Dev3.Web.RateLimit do
+  import Phoenix.Controller, only: [json: 2]
+  import Plug.Conn, only: [put_status: 2, halt: 1, send_resp: 3]
+  import Logger
+
+  @slack_messenger Application.get_env(:dev3, :slack_messenger)
+
+  def rate_limit(conn, options \\ []) do
```

Comment:
Why `options` is defaulted to an empty list ? Shouldn't it be an empty map instead ?